sechub-integrationtest/src/test/java/com/mercedesbenz/sechub/integrationtest/scenario3/ProjectAdministrationScenario3IntTest.java

```diff
@@ -91,6 +97,22 @@ void change_project_ownership_by_admin_and_owners() {
             isNotAssignedToProject(PROJECT_2). // still assigned after ownership loss
             hasOwnerRole(); // has still owner role
 
+        /* test 4 - project details contain profile IDs */
+
+        // normal user can view profiles of assigned projects
+        List<ProjectData> projectDetailsOfnormalUser = as(USER_2).getAssignedProjectDataList();
+        assertThat(projectDetailsOfnormalUser).hasSize(1);
+        Set<String> userAssignedProfileIds = projectDetailsOfnormalUser.get(0).getAssignedProfileIds();
+        assertThat(userAssignedProfileIds).containsExactly(PROFILE_1.id);
+
+        // owner can view profiles of assigned projects
+        List<ProjectData> projectDetailsOfOwner = as(USER_1).getAssignedProjectDataList();
+        assertThat(projectDetailsOfOwner).hasSize(1);
+        Set<String> ownerAssignedProfileIds = projectDetailsOfOwner.get(0).getAssignedProfileIds();
+        assertThat(ownerAssignedProfileIds).containsExactly(PROFILE_1.id);
+
+        // admin can view profiles of assigned projects
```

Comment:
first of all the comment is not correct:
It would be more:
```suggestion
        // admin has no assigned projects
```

But... this is problematic/critical:
All users of the scenario are specific for test and will automatically reset - except the SUPER_ADMIN: this user is always existing, which means: If there is ANY other test executed before, which would assign the super admin to a project, this test wold fail!

If your assumption is here, that an admin is not automatically assigned to projects, then you would need to create here another user and set him to an administrator.
Or you could check here, that the SUPER_ADMIN does not contain PROFILE_1.id

=> for next loop (even when its empty, this works) and check that this does not happen.

Tests are not executed always in same order by gradle or inside IDEs - so I would prefer if we change this to avoid to become this a flaky test in future.